### PR TITLE
Add `pallet-bridge-grandpa`, `pallet-bridge-parachains` to Pangolin

### PR DESCRIPTION
**Pull request by furoxr**
CC @HackFisher @AurevoirXavier @AsceticBear 

- pallet-bridge-grandpa
- pallet-bridge-parachains

### CODE REVIEW

File: node/runtime/pangolin/src/pallets/bridge_rococo_grandpa.rs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	// Assuming the worst case of every header being finalized, we will keep headers for at least a
+	// week.
+	pub const HeadersToKeep: u32 = 7 * DAYS as u32;
+}
+
+impl Config<WithRococoGrandpa> for Runtime {
+	type BridgedChain = Rococo;
+	type MaxRequests = MaxRequests;
+	type HeadersToKeep = HeadersToKeep;
+	type WeightInfo = ();
```
Comment (HackFisher) [inline node/runtime/pangolin/src/pallets/bridge_rococo_grandpa.rs:27]: @AurevoirXavier How to config the WeightInfo? Are we OK to set `()`?

Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/bridge_rococo_grandpa.rs:27]: `()` means default.

Before we bench this pallet, I think we can use the default value.

File: node/runtime/pangolin/src/pallets/bridge_pangolin_parachain.rs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+use pallet_bridge_grandpa::Instance2 as RococoGrandpaInstance;
+
+
```
Comment (AsceticBear) [inline node/runtime/pangolin/src/pallets/bridge_pangolin_parachain.rs:3]: Empty Line

**Comment by AurevoirXavier** [comment]
After the local testing finished. I'll review this.

**Comment by AurevoirXavier** [comment]
I made some changes. Please check.

**Pull request merged**